torchqdynamics/solver_utils.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+from math import sqrt
```

Comment:
When possible I think we should favour torch functions.

Reply:
`torch.sqrt` does not support floats

```
In [1]: import torch

In [2]: torch.sqrt(4.0)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[2], line 1
----> 1 torch.sqrt(4.0)

TypeError: sqrt(): argument 'input' (position 1) must be Tensor, not float
```

Reply:
Ho ok haha, sorry.